client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -636,6 +645,13 @@ export const ApplicationsTableAnalyze: React.FC = () => {
                       <TagIcon />
                       {application.tags ? application.tags.length : 0}
                     </Td>
+                    <Td
+                      width={10}
+                      modifier="truncate"
+                      {...getTdProps({ columnKey: "effort" })}
+                    >
+                      {application.effort ? application.effort : "-"}
```

Comment:
```suggestion
                      {application?.effort ?? "-"}
```
...the optional chaining with nullish coalescing way

Reply:
Very nice.